src/pages/Animal/MyPets/index.tsx

```diff
@@ -57,7 +61,7 @@ const MyPets = (): JSX.Element => {
           fetchedPets.map((pet) => (
             <AnimalCard
               key={uuidv4()}
-              imageUrlPromise={storage().ref(`${Values.IMAGE_DIRECTORY}/${pet.pictures.length > 0 ? `${pet.pictures[0]}` : 'pet.jpg'}`).getDownloadURL()}
+              imageUrlPromise={animalAPI.animalStoragePictureDir().child(`${pet.pictures.length > 0 ? `${pet.pictures[0]}` : 'pet.jpg'}`).getDownloadURL()}
```

Comment:
The same as before: `Animals.getPictureById(pictureId)`

Reply:
Fixed by new commit.